libs/pyodide-sandbox-js/main.ts

```diff
@@ -263,20 +302,18 @@ async function runPython(
 
       // Load PKL file into pyodide if it exists
       try {
-        const sessionData = await Deno.readFile(sessionPklPath);
-        pyodide.FS.writeFile(`/${options.session}.pkl`, sessionData);
+        sessionData = await Deno.readFile(sessionPklPath);
       } catch (error) {
         // File doesn't exist or can't be read, skip loading
       }
     }
 
     // Import our prepared environment module
     const prepare_env = pyodide.pyimport("prepare_env");
-    
-    // Prepare additional packages to install (include dill if using sessions)
-    const additionalPackagesToInstall = options.session
+    // Prepare additional packages to install (include dill)
+    const additionalPackagesToInstall = options.session || options.sessionBytes
       ? [...new Set([...sessionMetadata.packages, "dill"])]
-      : [];
+      : ["dill"];
```

Comment:
can remove dill if there's no session?

Reply:
we should probably have a flag for this (i.e. control whether to output session bytes). currently it was always outputting